NBXplorer.Tests/UnitTest1.cs

```diff
@@ -777,7 +777,8 @@ private static void CanCreatePSBTCore(ServerTester tester, ScriptPubKeyType type
 			Assert.Equal(outpoints[0], actualOutpoints[0]);
 			request.MinValue = Money.Coins(0.1m);
 			ex = Assert.Throws<NBXplorerException>(() => tester.Client.CreatePSBT(userDerivationScheme, request));
-			Assert.Equal("not-enough-funds", ex.Error.Code);
+			Assert.Equal("output-too-small", ex.Error.Code);
```

Comment:
this seems wrong to me.
`MinValue` is restricting the selected UTXOs. So the error message should be `not-enough-funds`, not `output-too-small`.

Reply:
So I looked over this case and while this is a breaking change, this isn't really illogical.

The PSBT is a "SweepAll", but there isn't any coin left in the wallet after the `MinCoins` set above.
So "Send all to address X" is basically "Send 0 to address X" which is indeed an "output that is too small".

Reply:
I will fix this PR to keep compatibility with the previous behavior, as it makes sense to say that there isn't enough funds to sweep all.